stripe.php

```diff
@@ -112,6 +112,28 @@ function stripe_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
 }
 
 /**
+ * Implementation of hook_civicrm_validateForm().
+ *
+ * Prevent server validation of cc fields
+ *
+ * @param $formName - the name of the form
+ * @param $fields - Array of name value pairs for all 'POST'ed form values
+ * @param $files - Array of file properties as sent by PHP POST protocol
+ * @param $form - reference to the form object
+ * @param $errors - Reference to the errors array.
+ */
+function stripe_civicrm_validateForm($formName, &$fields, &$files, &$form, &$errors) {
+  if (isset($form->_paymentProcessor['payment_processor_type']) &&$form->_paymentProcessor['payment_processor_type'] == 'Stripe') { 
```

Comment:
Well…we could probably just use the existence of the stripe token if $form->_paymentProcessor['payment_processor_type'] is not there.  I think that would preserve the ability to switch payment processors on front end forms while making the backend forms work.  It probably is not very likely that switching processors would be a problem with the backend forms anyway.

I think it would be possible to grab the stipe id and store it in the backend forms.  I did that with the front end forms but used $form->_paymentProcessor to do it and it doesn't look like that will work for backend forms. I would need to think about this a bit. I guess we could always query the database to get it.